packages/react/src/common_api_types.ts

```diff
@@ -0,0 +1,29 @@
+export type EntityKind = 'individual' | 'business'
+
+export type Location = {
+  address_one: string
+  address_two: string | null
+  city: string
+  region: string | null
+  postal_code: string
+  state: string | null
+  country: string
+}
+
+export type UserSession = {
+  maybe_anonymizing_proxy: boolean
+  country_code: string | null
+  region: string | null
+  city: string | null
+}
+
+export type BusinessType =
+  | 'Public Charity'
+  | 'Private Foundation'
+  | 'S Corporation'
+  | 'C Corporation'
+  | 'Irrevocable Trust'
+  | 'Revocable Trust'
+  | 'Partnership LLC'
+  | 'Partnership LP'
+  | 'Family Office'
```

Comment:
```suggestion
  | 'Family Office'
  | 'Limited Liability Company'
  | 'Limited Partnership'
```

These will be new values as of Feb 7, so might as well include them now?

Reply:
Awesome, thanks!